stimela/commands/run.py

```diff
@@ -1,22 +1,25 @@
 from collections import OrderedDict
 import dataclasses
 import itertools
+from datetime import datetime
 
 from yaml.error import YAMLError
 from scabha import configuratt
 from scabha.exceptions import ScabhaBaseException
 from omegaconf.omegaconf import OmegaConf, OmegaConfBaseException
+from stimela import stimelogging
 from stimela.config import ConfigExceptionTypes
 import click
 import logging
-import os.path, yaml, sys
+import os.path, yaml, sys, asyncio
```

Comment:
Not kosher. Unless importing from the same module/library, each import should be on a new line

Reply:
PEP-8 says "Imports should *usually* be on separate lines", so I tend to interpret that loosely. Particularly when a large number of the usual standard library suspects are imported, like ``import os, os.path, sys`` etc. I just find the import section gets too big.